integration-cli/docker_cli_kill_test.go

```diff
@@ -42,9 +42,11 @@ func (s *DockerSuite) TestKillDifferentUserContainer(c *check.C) {
 	dockerCmd(c, "kill", cleanedContainerID)
 
 	out, _ = dockerCmd(c, "ps", "-q")
-	if strings.Contains(out, cleanedContainerID) {
-		c.Fatal("killed container is still running")
-	}
+	processContainerID := strings.TrimSpace(out)
+
+	c.Assert(waitRun(processContainerID), check.IsNil)
```

Comment:
okay, so should I remove the if..else and the c.Assert statements?  Or is it better to leave it as an if..else statement.  I could also rewrite c.Assert statement?  

If I am understanding the test correctly, it checks to see that the containers are killed properly.  And at these lines its making sue that the containers are not running anymore.  Is it expected that the container is supposed to still be running?